FV3/conf/modules.fv3.daint_gnu

```diff
@@ -0,0 +1,12 @@
+##
+## load programming environment
+## this typically includes compiler, MPI and job scheduler
+##
+module load daint-gpu
```

Comment:
are we assuming anytime we run the model on daint we are running on a gpu? Or, probably more likely, loading the module on a cpu doesn't cause any problems? I thought our baseline was going to be made on cpu since that's what it's optimized for (though it running on gpu is useful to know as well)?

Reply:
We have an allocation on the GPU partition. So even if we run on CPU, we will be running on the CPUs of the GPU partition (Haswell).